lib/GridListImageItem/GridListImageItem.js

```diff
@@ -191,6 +191,7 @@ module.exports = kind(
 	* @private
 	*/
 	imageSizingChanged: function () {
+		this.$.image.addRemoveClass('enyo-gridlist-imageitem-image', (!this.imageSizing && !!this.placeholder));
```

Comment:
Hi @rajyavardhanp, if you take a look here, you will see that we were not intending to allow the image to have a natural size: https://github.com/enyojs/layout/blob/2.6.0-dev/src/GridListImageItem/GridListImageItem.css#L13.